walsamlee/politico
# #163416795 Add Edit Party by ID Endpoint

#### What does this PR do?
Add `edit-party-by-ID` API endpoint
#### Description of Task to be completed?
Have `/api/v1/parties/:partyId/:name` endpoint working
#### How should this be manually tested?
- Clone the project, https://github.com/walsamlee/politico
- Check out into the proper branch, `ft-v1-edit-party-163416795`
- Using postman, `PATCH` with API endpoint URL `/api/v1/parties/:partyId/:name` 
#### What are the relevant pivotal tracker stories?
#163416795
